state/upgrades_test.go

```diff
@@ -16,14 +16,13 @@ import (
 	"github.com/juju/utils/set"
 	"github.com/kr/pretty"
 	gc "gopkg.in/check.v1"
-	charm "gopkg.in/juju/charm.v6"
+	"gopkg.in/juju/charm.v6"
```

Comment:
so I realize it is redundant, but I also find that developers don't always make the association that the name that is actually created locally is whatever "package BLAH" definition is found at the associated path, and putting the name here can make it clearer.
We don't have a standard pattern of always naming them, or never naming them, but I generally don't touch them. Do you feel we should be more consistent here?

Reply:
I just generally revert anything non-vanilla that the linter tells me is redundant.

I am happy to let them alone going forward.